lint-staged.config.js

```diff
@@ -0,0 +1,19 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: loopback-next
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+const path = require('path');
+
+module.exports = {
+  '*.{js,ts,md}': files => {
+    const cwd = process.cwd();
+    files = files.map(f => path.relative(cwd, f));
+    const commands = [
+      `node packages/build/bin/run-prettier --write ${files.join(' ')}`,
+      `node packages/build/bin/run-eslint --report-unused-disable-directives` +
+        ` --fix --cache ${files.join(' ')}`,
```

Comment:
Please note that with TypeScript-powered linter checks, it's not enough to lint the staged files. For example, when the signature of a method inside `@loopback/context` changes (e.g. a `void` function is returning a `Promise` now), we need to lint all packages using that particular method. Many of them will not pass [no-floating-promises](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-floating-promises.md) check.